bin/supply

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+echo "-----> VulsBuildpack/supply"
+
+BIN_DIR=$(cd $(dirname $0); pwd)
+ROOT_DIR=$(dirname $BIN_DIR)
+BUILD_DIR=$1
+CACHE_DIR=$2
+DEPS_DIR=$3
+INDEX=$4
+
+VULS_URL="${VULS_URL:-https://github.com/future-architect/vuls/releases/download/v0.9.6/vuls_0.9.6_linux_amd64.tar.gz}"
+VULS_DIR=$DEPS_DIR/$INDEX
+
+echo "Installing the vuls to ${VULS_DIR}"
+
+mkdir -p $VULS_DIR
+mkdir -p $BUILD_DIR/.profile.d
+
+cp -r $ROOT_DIR/lib/dist $VULS_DIR/
+
+echo "Download vuls asset from ${VULS_URL}"
+curl -L -o $VULS_DIR/vuls.tar.gz $VULS_URL
```

Comment:
pull `vuls.tar.gz` into a variable as it is repeated on L25 & L27 below

Reply:
Sounds like a good plan, will do.

Reply:
Added.